packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -303,17 +303,17 @@ export default class ContentScript {
    * Wait for the given labelled request to be intercepted. The labeled request must be defined and
```

Comment:
Replace labelled with identified

Reply:
Oopsi, indeed.

There's is something else with label, the build won't work, i'm looking for it 👍